docs/customservices.md

```diff
@@ -272,4 +272,9 @@ Configuration example:
   warning_value: 50
   danger_value: 80
   api_token: "PVEAPIToken=root@pam!your-api-token-name=your-api-token-key"
+  # values below this line are optional (default value are false/empty):
+  hide_decimals: true # removes decimals from stats values.
+  hide: ["vms", "vms_total", "lxcs", "lxcs_total", "disk", "mem", "cpu"] # hides values included in the array
+  small_font_on_small_screens: true # uses small font on small screens (like mobile)
+  small_font_on_desktop: true # uses small font on desktops (just in case you're showing much info)
```

Comment:
Not sure about this one either, it could be automatic based on the number of items choosen? :thinking:

Reply:
Same thing, I was thinking about big screen sizes (ultra wide monitors). Right now, max wide is a fixed value, but I was willing to try and make a PR allowing that to be changed :)

If you feel like you're gonna keep the max wide fixed, I can make this one automatic according to which fields are shown.

Reply:
I've been playing around to see the edge cases of setting font size automatically **only on desktop** and there are several things involved:

- If I `hide_decimals` and `*_total`, the font is good at the default size.
- If I don't `hide_decimals` but hide `*_total` small font is needed.
- If I hide any other field, default font size is great.

I can write this logic to make this automatic, is not too hard as I build and array with fields to hide (as vms and lxcs are hidden by default if none of them exist in proxmox), but not sure if that's the best way to go.

Reply:
I will recover this PR as the old one and merge the main branch to resolve conflicts. Do you wanto to go deeper into this or should we resolve this conversation?

If in the future you add a custom class to be used, I can update this to the newer styles.